pennylane/ops/op_math/linear_combination.py

```diff
@@ -40,6 +40,11 @@ class LinearCombination(Sum):
         simplify (bool): Specifies whether the ``LinearCombination`` is simplified upon initialization
                          (like-terms are combined). The default value is `False`. Note that ``coeffs`` cannot
                          be differentiated when using the ``'torch'`` interface and ``simplify=True``.
+
+                            .. warning::
+                                The ``simplify`` argument is deprecated and will be removed in a future release.
+                                Instead, you can call ``qml.simplify`` on the constructed operator.
+
```

Comment:
This isn't looking right in the docs: 
![image](https://github.com/PennyLaneAI/pennylane/assets/49409390/d016439e-9ac6-41e9-9eb4-e48a85b275fe)

Reply:
Yep, there is a discussion with @albi3ro about this on Slack. If we put a warning message under an argument in the doc, Sphinx inserts this annoying blue box. I tried many other indentations, but this messes up the remaining description of the argument for some reason. We were undecided whether to keep this version or to insert the `warning` message under the description of all the arguments, with a **DEPRECATED** message at the beginning of the `simplify` argument. What do you think @trbromley ?